<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>591</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>591</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
